api/auth/index.test.js

```diff
@@ -24,23 +28,33 @@ tap.test('authentication setup', async authTest => {
     authenticate: sandbox.stub().returns('passport-authenticate')
   };
 
+  const session = sandbox.stub();
+
   const strategies = ['strategy1', 'strategy2'];
 
+  const res = {
+    send: sandbox.stub(),
+    status: sandbox.stub(),
+    end: sandbox.spy()
+  };
+  res.send.returns(res);
+  res.status.returns(res);
```

Comment:
Moved `res` higher up in the scope, since it was being declared in multiple places.  I needed to add another place, so it made sense to just bump it up here.